ios/FluentUI/Vnext/List/HeaderFooterTokens.swift

```diff
@@ -8,61 +8,58 @@ import SwiftUI
 
 /// HeaderFooter styles
 @objc public enum MSFHeaderFooterStyle: Int, CaseIterable {
-    case headerPrimary
-    case headerSecondary
+    case standard
+    case subtle
 }
 
-class MSFHeaderFooterTokens: MSFTokensBase, ObservableObject {
-    @Published public var backgroundColor: UIColor!
-    @Published public var textColor: UIColor!
+open class MSFHeaderFooterTokens: ControlTokens {
+    /// The MSFHeaderFooter style. Currently we only have primary and secondary styles.
```

Comment:
I'm not sure we need to list out which styles we have available, but we have standard and subtle, not primary and secondary.